### PR TITLE
Clean up ceph variables

### PR DESCRIPTION
**Pull request by msambol**

### CODE REVIEW

File: roles/cinder-common/defaults/main.yml [hunk 1]
```diff
@@ -51,3 +54,10 @@ cinder:
     debug: False
     verbose: True
   cafile: "{{ ssl.cafile|default('/etc/ssl/certs/ca-certificates.crt') }}"
+  ceph:
+    rbd_pool: default
```
Comment (j2sol) [inline roles/cinder-common/defaults/main.yml]: Does this name have to change if the `ceph.pool_name` changed?

File: roles/cinder-common/defaults/main.yml [hunk 2]
```diff
@@ -51,3 +54,10 @@ cinder:
...
+    rbd_user: cinder
```
Comment (j2sol) [inline roles/cinder-common/defaults/main.yml]: Same with this

**Comment by j2sol** [comment]
I have questions on data duplication across multiple roles.

File: roles/cinder-common/defaults/main.yml [hunk 3]
```diff
@@ -51,3 +54,9 @@ cinder:
     debug: False
     verbose: True
   cafile: "{{ ssl.cafile|default('/etc/ssl/certs/ca-certificates.crt') }}"
+  ceph:
+    rbd_user: cinder
+    rbd_ceph_conf: /etc/ceph/ceph.conf
+    rbd_flatten_volume_from_snapshot: false
+    rbd_max_clone_depth: 5
+    glance_api_version: 2
```
Comment (msambol) [inline roles/cinder-common/defaults/main.yml]: @j2sol I'd rather move all these into `ceph-common/defaults/main.yml`, even though they are used by the cinder role. Thoughts?

**Comment by msambol** [comment]
This clears up data duplication. `ceph-compute` has no variables defined. `ceph-common` variables are moved into `ceph` dict.

File: roles/cinder-common/defaults/main.yml (hunk 3 above)
Comment (j2sol) [inline roles/cinder-common/defaults/main.yml]: Well... Yeah I could see these moving, and anything glance related too, then the glance role can just reference `ceph.whatever`.

Comment (j2sol) [inline roles/cinder-common/defaults/main.yml]: Yeah, glance needs to know the rbd_ceph_conf too, although it isn't currently templated, just hard coded. Oops.

**Comment by j2sol** [comment]
:+1:

**Pull request merged**